### PR TITLE
Build v3 utility classes: Margins, Measure, and Padding

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Chromatic

https://1031-v3-utilities--60f9b557105290003b387cd5.chromatic.com

## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1031

## Acceptance criteria
- [x] The margins, measure, and padding utility classes based on USWDS v3 have been added to the new CSS library

## Definition of done
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/css-library/src/tokens/measure.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$tokens-measure: (
```
Comment (k80bowman) [inline packages/css-library/src/tokens/measure.scss:1]: I wonder if this name should be a little more descriptive.

Comment (k80bowman) [reply packages/css-library/src/tokens/measure.scss:1]: Nevermind, I see now that USWDS also uses this terminology. I don't care for it, but probably better to stay in sync with them on this.

File: packages/css-library/src/utilities/margin.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+$u-margin: map-collect(
+  $u-margin-base,
+  $u-margin-y,
+  $u-margin-top,
+  $u-margin-bottom,
+  $u-margin-x,
+  $u-margin-right,
+  $u-margin-left
+);
```
[the remark this answers is not in the document]
Comment (andresriveratoro) [reply packages/css-library/src/utilities/margin.scss:96]: The negative margin tokens were used for $u-margin-y, $u-margin-top, $u-margin-bottom, $u-margin-x, $u-margin-right and $u-margin-left.

```
$tokens-margin-y: map-collect($tokens-margin, $tokens-negative-margin);

$tokens-margin-top: map-collect($tokens-margin, $tokens-negative-margin);

$tokens-margin-bottom: map-collect($tokens-margin, $tokens-negative-margin);

$tokens-margin-x: map-collect(
  $tokens-margin,
  $tokens-negative-margin,
  (
    auto: -1px,
  )
);

$tokens-margin-right: map-collect(
  $tokens-margin,
  $tokens-negative-margin,
  (
    auto: -1px,
  )
);

$tokens-margin-left: map-collect(
  $tokens-margin,
  $tokens-negative-margin,
  (
    auto: -1px,
  )
);
```

[the remark this answers is not in the document]
Comment (k80bowman) [reply packages/css-library/src/utilities/margin.scss:96]: ah so I was missing something then. Thank you for clarifying!

File: packages/css-library/src/tokens/margin.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+$tokens-margin-top: map-collect($tokens-margin, $tokens-negative-margin);
+
+$tokens-margin-bottom: map-collect($tokens-margin, $tokens-negative-margin);
+
+$tokens-margin-x: map-collect(
+  $tokens-margin,
+  $tokens-negative-margin,
+  (
+    auto: -1px,
```
Comment (bkjohnson) [inline packages/css-library/src/tokens/margin.scss:49]: Why do we have this on margins `x`, `right`, and `left`, but not `y`, `top`, and `bottom`?

File: packages/css-library/src/utilities/margin.scss [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+$u-margin-bottom: (
+  margin-bottom: (
+    base: 'vads-margin-bottom',
+    modifiers: null,
+    values: map-collect($tokens-margin-bottom),
+    settings: $margin-settings,
+    property: 'margin-bottom',
+    type: 'utility',
+  ),
+);
```
Comment (bkjohnson) [inline packages/css-library/src/utilities/margin.scss:50]: Could we use the `modifiers` for these like we're doing in the `padding` utilities?

File: packages/css-library/src/utilities/padding.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+$u-padding-base: (
+  padding: (
+    base: 'vads-padding',
+    modifiers: (
+      noModifier: '',
+      'top': '-top',
+      'bottom': '-bottom',
+      'right': '-right',
+      'left': '-left',
+    ),
```
Comment (bkjohnson) [inline packages/css-library/src/utilities/padding.scss:14]: Re: previous comment.

File: packages/css-library/src/tokens/margin.scss (hunk 3 above)
Comment (andresriveratoro) [reply packages/css-library/src/tokens/margin.scss:49]: The auto utility exists for left, right and x, but not for the other [margin utilities](https://design.va.gov/foundation/utilities/margins).
Screen Shot 2022-08-23 at 11 52 58
Screen Shot 2022-08-23 at 11 53 13
Screen Shot 2022-08-23 at 11 53 08

File: packages/css-library/src/utilities/margin.scss (hunk 4 above)
Comment (andresriveratoro) [reply packages/css-library/src/utilities/margin.scss:50]: We could use it for the utilities that have the same map values, but not for all margin utilities. I'll consolidate the ones we can. 👍

**Review by bkjohnson — APPROVED** [review]
Looks good to me.

**Pull request merged**